dimod/binary_quadratic_model.py

```diff
@@ -224,6 +238,11 @@ def empty(cls, vartype):
                 * :attr:`.Vartype.SPIN`, ``'SPIN'``, ``{-1, 1}``
                 * :attr:`.Vartype.BINARY`, ``'BINARY'``, ``{0, 1}``
 
+            dtype (type/function, optional, default=object):
```

Comment:
Just out of curiosity, `empty(cls, vartype, dtype=object)` creates a BQM with no bias values, correct? Is the `dtype` here for the future, when the user decides to add in bias values later on?

Reply:
That's correct. The dtype is a property of the bqm as a whole, so future biases added are coerced to the appropriate type.